tinygrad/mlops.py

```diff
@@ -155,6 +155,14 @@ def forward(self, x, axis):
   def backward(self, grad_output):
     return grad_output.movement_op(MovementOps.FLIP, self.axis)
 
+class Stride(Function):
+  def forward(self, x, stride):
+    self.stride = stride
+    return x.movement_op(MovementOps.STRIDED, stride)
+
+  def backward(self, grad_output):
+    return grad_output.movement_op(MovementOps.STRIDED, self.stride)
```

Comment:
Hmm, is this the derivative of STRIDED?

Reply:
No, I pushed before checking the backward pass properly...

I'll have a go at fixing it!